theme_monglia/views/customizations.xml

```diff
@@ -148,6 +148,14 @@
     </xpath>
 </template>
 
+<!-- ======== IMAGE HEXAGONAL ======== -->
+<template id="s_image_hexagonal" inherit_id="website.s_image_hexagonal">
+    <!-- Images -->
+    <xpath expr="(//img)[2]" position="attributes">
+        <attribute name="src">/web/image/website.s_text_cover_default_image</attribute>
```

Comment:
Not sure about this ? Did you really want to use that image here?

Reply:
I think it's a cache issue on your side. It should be this image

<img width="481" alt="Screenshot 2024-08-21 at 17 29 55" src="https://github.com/user-attachments/assets/cf65ba9d-53ca-4fd7-9da3-01944f8f1184">